blocks/video-promo-block/features/video-promo/default.jsx

```diff
@@ -28,26 +30,40 @@ const AlertBadge = styled.span`
 const VideoPromo = ({ customFields }) => {
   const { arcSite } = useFusionContext();
   const {
-    uuid,
     autoplay = false,
-    ratio = 0.5625,
-    title,
-    description,
+    inheritGlobalContent = true,
     playthrough = false,
     alertBadge,
   } = customFields;
 
-  const content = uuid ? null : useContent({
+  // can be overwrite by globalContent
+  let {
+    title,
+    description,
+  } = customFields;
+
+  const { globalContent = {} } = useFusionContext();
+
+  const content = inheritGlobalContent ? globalContent : useContent({
```

Comment:
can you try to fix this conditional `useContent`? we talked about trying not to add more react-hooks warning in the meeting. see here for more info https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
updated & test coverage updates
<img width="1127" alt="Screen Shot 2020-11-19 at 12 10 51 AM" src="https://user-images.githubusercontent.com/4411687/99638900-ca26c280-29fb-11eb-9db4-1abaa19f56d5.png">